.pre-commit-config.yaml

```diff
@@ -7,5 +7,5 @@ repos:
 - hooks:
   - id: flake8
     language_version: python3
-  repo: https://gitlab.com/pycqa/flake8
-  rev: 4.0.1
+  repo: https://github.com/PyCQA/flake8
+  rev: 5.0.4
```

Comment:
This is obviously unrelated to pyinstaller, but we should be using the github repo for flake8 (the old gitlab URL now requires auth and breaks pre-commit in CI)